src/functions/utils/dailyTax.ts

```diff
@@ -0,0 +1,14 @@
+import { IInformationsFromJsonFile } from '../../Interface/IInformationsFromJsonFIle';
+import { countTaxableProfit } from './countTaxableProfit';
+import { taxesToPay } from './taxesToPay';
+
+export function dailyTax(taxPaid: number[], tips: number[], budget: number[], tipsTax: number, dailyTax: number): { dailyTaxAmount: number; tipsTaxToPay: number } {
+    const tipsTaxFromJson = tipsTax != undefined ? tipsTax : 5;
```

Comment:
Is it possible to assign `undefined` to `number`? (im not sure)

Possibly you want to have input validation on some level (the closer to input the better), but on some level you probably would prefer to KNOW that input is valid and you can work with it.
I suggest, that this `level` should be around mapping json file into IInformationsFromJsonFile object.
You should properly validate properties and possibly override unwated values with defaults.
You can talk with Żółw about this.

Goal is to not have to check `tipsTax != undefined ? tipsTax : defaultValue` in EVERY METHOD IN SYSTEM